srml/aura/src/lib.rs

```diff
@@ -14,7 +14,60 @@
 // You should have received a copy of the GNU General Public License
 // along with Substrate.  If not, see <http://www.gnu.org/licenses/>.
 
-//! Consensus extension module for Aura consensus. This manages offline reporting.
+//! ## Overview
+//! 
+//! The `aura` module extends Aura consensus by managing offline reporting.
+//! 
+//! ## Interface
+//! 
+//! ### Dispatchable
+//! 
+//! There are no dispatchable functions for this module as it mainly serves to monitor offline interaction in the context of Aura consensus and doesn't operate independent of this consensus algorithm.
+//! 
+//! ### Public
+//! 
+//! See the [module](https://crates.parity.io/srml_aura/struct.Module.html) for details on publicly available functions.
+//! 
+//! **Note:** When using the publicly exposed functions, you (the runtime developer) are responsible for implementing any necessary checks (e.g. that the sender is the signer) before calling a function that will affect storage.
```

Comment:
true, but I'm not sure it worth saying it there, I think it is told in decl_module doc
is it part of modules doc template ? I could agree to keep it

Reply:
fair, I've just seen it in other module doc, but it isn't really necessary for this one...I'll take it out